src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js

```diff
@@ -18,6 +21,44 @@ kunstmaanbundles.autoCollapseTabs = (function($, window, undefined) {
         }); // when window is resized
     };
 
+    initTabLogic = function () {
+        // If there is a tab defined in the url, we activate it
+        if ($('#currenttab') && $('#currenttab').val().length > 0) {
```

Comment:
Maybe save $('#currenttab') to a var so it does not have to be re-selected all the time.